### PR TITLE
Updates scope-extensions explainer

### PR DESCRIPTION
**Pull request by diekus**
* adds details for scope extensions explainer

@dmurph @LuHuangMSFT fixed conflicts. worked on an old branch :S

### CODE REVIEW

File: scope_extensions-explainer.md [hunk 1]
```diff
@@ -7,6 +7,26 @@ web apps to extend their
...
+
+A web application's content might originate from different scopes. Currently, if
+an installed web application navigates to a url that is out of the scope defined
+in the manifest file, a security UX will appear in the form of a bar to indicate
+to the user that they are outside of the defined scope of the application. See
+figure below. 
+
+![Installed web app showing out-of-scope UX](images/out-of-scope-ux.png)
+
+The app in he image (`PWinter`) has navigated to a url out of its scope
```
Comment (LuHuangMSFT) [inline scope_extensions-explainer.md]:
```suggestion
The app in the image (`PWinter`) has navigated to a url out of its scope
```

File: scope_extensions-explainer.md [hunk 2]
```diff
@@ -7,6 +7,26 @@ web apps to extend their
...
+(`airhorner.com`). The white bar on top of the web app is providing information
+to the user about this change of scope. While this is a security feature, it can
+be the case that an application would want to extend its scope. For example, an
```
Comment (LuHuangMSFT) [inline scope_extensions-explainer.md]:
```suggestion
be the case that an application wants to extend its scope. For example, an
```

File: scope_extensions-explainer.md [hunk 3]
```diff
@@ -7,6 +7,26 @@ web apps to extend their
...
+application might host content that is located in one specific origin, and rely
+on a login page that is out of the scope of the application itself to access
+that content. In other cases, the same application might be associated to
+multiple Top Level Domains, that might respond to different geographies
```
Comment (LuHuangMSFT) [inline scope_extensions-explainer.md]:
```suggestion
multiple Top Level Domains, that might respond to different locales
```

File: scope_extensions-explainer.md [hunk 4]
```diff
@@ -160,13 +180,33 @@ the following attack vector:
...
+included in a scope is that the security UX will not appear when an app
+navigates to content served from those scopes.
+
+### Additional security UX
+
+For added security when in the installed web application, the app might display
+UX that always displays the current scope that is being served, along with
+privacy and permission settings of that specific scope.
+
+![Current scope being displayed on the privacy menu of installe web
```
Comment (LuHuangMSFT) [inline scope_extensions-explainer.md]:
```suggestion
![Current scope being displayed on the privacy menu of installed web
```

**Review by LuHuangMSFT — APPROVED** [review]
Approved with suggested changes.

**Review by dmurph — APPROVED** [review]

**Pull request merged**